src/ConfigFactory.Core/ConfigModule.cs

```diff
@@ -112,13 +112,11 @@ public virtual void Save()
         }
     }
 
-    public virtual bool Validate() => Validate(out _, out _);
-    public virtual bool Validate(out string? message) => Validate(out message, out _);
     public virtual bool Validate(out string? message, out ConfigProperty target)
     {
-        foreach ((var name, (var validate, var errorMessage)) in Validators) {
+        foreach (var (name, (validate, errorMessage)) in Validators) {
             target = Properties[name];
-            PropertyInfo propertyInfo = target.Property;
+            var propertyInfo = target.Property;
```

Comment:
Prefer explicit type names

```suggestion
            PropertyInfo propertyInfo = target.Property;
```

Reply:
Sure, no problem :)